source/2020-03-27-the-ember-times-issue-141.md

```diff
@@ -45,19 +46,19 @@ So be on the look out 👀 for more in this very helpful new series!
 
 ---
 
-## [Achieve consistent decorator positioning with a new eslint plugin ↕️](https://github.com/NullVoxPopuli/eslint-plugin-decorator-position)
+## [A quick tip for refactoring your templates 📑](https://abhilashlr.in/ember-refactor-templates)
 
-Ever had to **put up with** inconsistent decorator positions in pull requests, because there wasn't a lint rule for that?
-_Now_ there is! 🎉 [eslint-plugin-decorator-position](https://github.com/NullVoxPopuli/eslint-plugin-decorator-position)
-provides some configuration with some recommended defaults to help your project achieve that last bit of consistency.
+[Abhilash L Ramesh @abhilashlr](https://github.com/abhilashlr) wrote a small and clear [blog post](https://abhilashlr.in/ember-refactor-templates) with actionable tips to make your Ember Templates cleaner. So if you have complex `if-else` logic in your template that you would like to make clearer, this blogpost could help you out. 
```

Comment:
It wasn't changed in this PR, but nevertheless I think it'd be

```suggestion
[Abhilash L Ramesh @abhilashlr](https://github.com/abhilashlr) wrote a small and clear [blog post](https://abhilashlr.in/ember-refactor-templates) with actionable tips to make your Ember Templates cleaner. So if you have complex `if-else` logic in your template that you would like to make clearer, this blog post could help you out. 
```

What do you think?

Reply:
Ah, good catch, thanks! I didn't check the other sections carefully.